plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/ExpEmbedding.kt

```diff
@@ -50,40 +57,45 @@ data class Add(
     override val left: ExpEmbedding,
     override val right: ExpEmbedding,
     override val source: KtSourceElement? = null,
+    override val info: Info = Info.NoInfo,
```

Comment:
That's true, I was unsure if I had to add it only on the nodes I'm interested into. But I've added to the `ExpEmbedding` for being consistent with other metadata, like the `source`.

Reply:
I'll be changing the source to work differently, actually, since it's a hassle to do it this way.  I would recommend adding `Info` just to the classes you currently need it in, and preferably as a post-construction hook, like a `withInfo` function that sets it.